vendor/github.com/raintank/schema/metric_test.go

```diff
@@ -0,0 +1,18 @@
+package schema
+
+import (
+	"testing"
+)
+
+func TestNameWithTagFromMetricData(t *testing.T) {
```

Comment:
i think we are probably doing a bit too much (in general, not just here) testing of particular implementations, rather than behavior.
that said i don't have a concrete suggestion, so ...

Reply:
Usually my reason to add tests like this one is just to verify that my implementation actually works. It wouldn't necessarily need to be committed, I could also just not commit those, but then somebody who wants to modify the implementation would have to test it again independently.